streamer.c

```diff
@@ -205,14 +205,10 @@ void cLiveStreamer::Action(void)
       // If some additional data is present inside the stream, it is written there (currently RDS inside MPEG2-Audio)
       if (pkt_side_data.data)
       {
-        if (pkt_side_data.streamChange || requestStreamChangeExtData)
+        if (pkt_side_data.streamChange || requestStreamChangeSideData)
           sendStreamChange();
```

Comment:
if it is not needed now, it should be removed. we should only add code which has any function by the time added